apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Properties/controls/ToggleOption/ToggleOption.stories.tsx

```diff
@@ -2,28 +2,27 @@ import { MockedProvider } from '@apollo/client/testing'
 import Typography from '@mui/material/Typography'
 import { Meta, StoryObj } from '@storybook/react'
 import noop from 'lodash/noop'
+import { ComponentPropsWithoutRef } from 'react'
 
 import { EditorProvider } from '@core/journeys/ui/EditorProvider'
 
 import { journeysAdminConfig } from '../../../../../../../../libs/storybook'
-import { Drawer } from '../../../../Drawer'
-
-import { ToggleOptionProps } from './ToggleOption'
 
 import { ToggleOption } from '.'
 
 const ToggleOptionStory: Meta<typeof ToggleOption> = {
   ...journeysAdminConfig,
   component: ToggleOption,
-  title: 'Journeys-Admin/Editor/ControlPanel/Attributes/ToggleOption'
+  title:
+    'Journeys-Admin/Editor/Slider/Settings/CanvasDetails/Properties/controls/ToggleOption'
 }
 
-const Template: StoryObj<typeof ToggleOption> = {
-  render: ({ ...args }: ToggleOptionProps) => {
+const Template: StoryObj<ComponentPropsWithoutRef<typeof ToggleOption>> = {
```

Comment:
Any reason for using `WithoutRef`?
```suggestion
const Template: StoryObj<ComponentProps<typeof ToggleOption>> = {
```

Reply:
Recommended in the ts doc to use the type with or without ref

Reply:
<img width="491" alt="image" src="https://github.com/JesusFilm/core/assets/91484241/37954235-0014-4c65-afea-9ba094e0eb82">